### PR TITLE
Calculate line angle start end

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**

### CODE REVIEW

File: src/components/network/layers/arrow-layer.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ const defaultProps = {
  * or with a parallel offset. The initial point is also shifted to coincide with the fork line ends.
  * Needs to be kept in sync with ForkLineLayer and ParallelPathLayer because they draw the lines.
  * props : getLineParallelIndex: accessor for real number representing the parallel translation, normalized to distanceBetweenLines
- *         getLineAngle: accessor for line angle in radian
+ *         getLineAngles: accessor for line angle in radian (3 angle substation1 / first pilone ; substation1/substation2 ; last pilone / substation2
```
Comment (jonenst) [inline src/components/network/layers/arrow-layer.js:56]: pilone -> pylon

File: src/components/network/layers/fork-line-layer.js [hunk 2]
```diff
@@ -32,21 +39,34 @@ export default class ForkLineLayer extends LineLayer {
...
-}
+
+    float offsetSubstation = clamp(project_size_to_pixel(substationRadius*instanceOffsetStart ), 
+                                    minSubstationRadiusPixel, 
+                                    substationMaxPixel * instanceOffsetStart );
+    float offsetSubstationCommonSpace = project_pixel_size(offsetSubstation) ;
+
+    vec4 trans = vec4(cos(instanceLineAngle), -sin(instanceLineAngle ), 0, 0.) * instanceLineParallelIndex;
+
+    if( instanceLineParallelIndex != 0. ){
```
Comment (jonenst) [inline src/components/network/layers/fork-line-layer.js:63]: Why do we need this special case ? Doesn't it lead to a mismatch with the arrow layer ?

File: src/components/network/layers/parallel-path-layer.js [hunk 3]
```diff
@@ -42,32 +42,45 @@ export default class ParallelPathLayer extends PathLayer {
...
-{
-  trans.x -= sin(instanceLineAngle);
-  trans.y -= cos(instanceLineAngle);
+float instanceLineParallelIndex = instanceParallelIndexAndProximityFactor[0];
+ 
+float offsetPixels = clamp(project_size_to_pixel(distanceBetweenLines), minParallelOffset, maxParallelOffset);
+float offsetCommonSpace = project_pixel_size(offsetPixels);
+vec4 trans = vec4(cos(instanceLineAngle), -sin(instanceLineAngle), 0, 0.) * instanceLineParallelIndex;
+
+if (instanceLineParallelIndex != 0. ){
```
Comment (jonenst) [inline src/components/network/layers/parallel-path-layer.js:72]: remove this if ?

Comment (JacquesBorsenberger) [reply src/components/network/layers/parallel-path-layer.js]: done

**Review by geofjamg — APPROVED** [review]

**Pull request merged**